plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/FirUtils.kt

```diff
@@ -24,3 +26,8 @@ val FirFunctionCall.functionCallArguments: List<FirExpression>
     get() = listOfNotNull(dispatchReceiver) + argumentList.arguments
 val FirFunctionSymbol<*>.effects: List<FirEffectDeclaration>
     get() = this.resolvedContractDescription?.effects ?: emptyList()
+val KtSourceElement?.asSilverPosition: Position
```

Comment:
Our convention is that `Silver` refers to the types in `viper.silver`, but here `Position` is our own class, so this should just be `asViperPosition` (or `asPosition`).